include/Jit/LLILCJit.h

```diff
@@ -19,11 +19,17 @@
 #include "Pal/LLILCPal.h"
 #include "options.h"
 #include "llvm/ExecutionEngine/ExecutionEngine.h"
+#include "llvm/ExecutionEngine/RuntimeDyld.h"
 #include "llvm/IR/LLVMContext.h"
 #include "llvm/Support/ManagedStatic.h"
 #include "llvm/Support/ThreadLocal.h"
 #include "llvm/ExecutionEngine/Orc/ObjectLinkingLayer.h"
 #include "llvm/ExecutionEngine/Orc/IRCompileLayer.h"
+#include "llvm/Config/config.h"
+#include "llvm/Object/ObjectFile.h"
+
+using namespace llvm;
+using namespace llvm::object;
```

Comment:
Nit: Per [the LLVM coding standards](http://llvm.org/docs/CodingStandards.html#do-not-use-using-namespace-std), I think it's best not to put using statements in headers:

> In header files, adding a `using namespace XXX` directive pollutes the namespace of any source file that #includes the header.